### PR TITLE
JUPITER-49, 47: CAD currency tooltip removed

### PR DESCRIPTION
**Pull request by anuritjain**
# Description
CAD currency updates as per [INTAKE-841](https://jira.rax.io/browse/INTAKE-841)
Jira Ids: [JUPITER-49](https://jira.rax.io/browse/JUPITER-49) & [JUPITER-47](https://jira.rax.io/browse/JUPITER-47)

image
### PR Checklist before asking for +2 from peers.
- [x] Did you include screenshots for changed UI functionality?
- [x] Did you add component tests for any new code?
- [x] Were you able to signup successfully with no console errors?
- [x] Have you deployed and validated your changes in staging?

### Before merging to master
- [x] I notified #jupiter-dev I am going to do a deployment before merging to master.

### CODE REVIEW

File: app/components/SignUp/BillingInfo/CurrencySelector.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ function CurrencySelector({ t, customerType, country, productType }) {
     {
       value: 'cad',
       label: t('account:billing.currency.cad'),
-      disabled: false
+      disabled: customerType === 'rbu'
```
Comment (abrande) [inline app/components/SignUp/BillingInfo/CurrencySelector.js:24]: Is CAD available now if they don't live in Canada?

File: i18n/en-US/account.json [hunk 2]
```diff
@@ -44,7 +44,7 @@
     "translation" : "For non-RBU AWS customers, you can select AUD."
   },
   "billing.tips.currency.cad.restrictions" : {
-    "translation" : "For Onica customers who live in Canada, you can select CAD."
+    "translation" : "For Rackspace & Onica customers, you can select CAD."
```
Comment (abrande) [inline i18n/en-US/account.json:47]: It looks like based on this ticket AWS, Azure, and GCP, Onica customers may select CAD. You can follow up with Chris Moyer if you'd like guidance on the wording. https://jira.rax.io/browse/INTAKE-841

**Review by abrande — APPROVED** [review]

File: i18n/en-US/account.json (hunk 2 above)
Comment (anuritjain) [reply i18n/en-US/account.json:47]: sure

Comment (nicko-winner) [reply i18n/en-US/account.json:47]: In last standup parking lot it was determined best to simply remove the tooltip. See comments in: https://jira.rax.io/browse/JUPITER-47

Comment (anuritjain) [reply i18n/en-US/account.json]: removed keys from i18n repo.

**Review by abrande — CHANGES_REQUESTED** [review]

File: app/components/SignUp/BillingInfo/CurrencySelector.js [hunk 3]
```diff
@@ -21,7 +20,7 @@ function CurrencySelector({ t, customerType, country, productType }) {
     {
       value: 'cad',
       label: t('account:billing.currency.cad'),
-      disabled: false
+      disabled: !cadAvailable
```
[the remark this answers is not in the document]
Comment (anuritjain) [reply app/components/SignUp/BillingInfo/CurrencySelector.js]: fixed

File: app/components/SignUp/BillingInfo/CurrencySelector.test.js [hunk 4]
```diff
@@ -57,14 +57,20 @@ describe('CurrencySelector', () => {
...
+      describe('when country is Canada', () => {
+        ['CA'].forEach((country) => {
+          const wrapper = mount({
+            country: 'CA'
+          });
+          const cadOption = wrapper.find('SelectorStrip').props().options[1];
+          test(`cad is enabled for ${country} customers`, () => {
+            expect(cadOption.disabled).toBeFalsy();
+          });
+        });
```
[the remark this answers is not in the document]
Comment (anuritjain) [reply app/components/SignUp/BillingInfo/CurrencySelector.test.js]: modified tests as mentioned.

**Review by abrande — APPROVED** [review]

**Review by nicko-winner — APPROVED** [review]

**Pull request merged**